portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +35,33 @@ function addRandomFact() {
   const randomFactContainer = document.getElementById('random-fact-container');
   randomFactContainer.innerText = fact;
 }
+
+/*
+  * Expands content when content title button is clicked.
+*/
+function toggleContentVisibility(){
```

Comment:
nit: fix comment styles.

Reply:
I think you may want to find an auto formatter tool so avoid this kind of issue.

Reply:
I have not tried any auto-formatter tool. Prettier.io seems to have different format than the style guide. Is there any tool you recommend to use to auto-format according to Google style guide?